changelogs/fragments/9600-apache2-mod-proxy-revamp2.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - apache2_mod_proxy - change type of ``state`` to ``list`` of ``str`` elements. No change for the users (https://github.com/ansible-collections/community.general/pull/9600).
```

Why not write it like this?
```suggestion
  - apache2_mod_proxy - change type of ``state`` to a list of strings. No change for the users (https://github.com/ansible-collections/community.general/pull/9600).
```